src/radical/pilot/pmgr/launching/default.py

```diff
@@ -1119,6 +1121,23 @@ def _prepare_pilot(self, resource, rcfg, pilot, expand):
             if virtenv_mode != 'private':
                 cleanup = cleanup.replace('v', '')
 
+        # don't stage sdists on 'installed', 'local'
+        if virtenv_mode in ['installed', 'local']: stage_sdists = False
+        else                                     : stage_sdists = True
+
+        # use local VE ?
+        if virtenv_mode == 'local':
+            if 'VIRTUAL_ENV' in os.environ:
+                virtenv_dist = 'default'
+                virtenv      = os.environ['VIRTUAL_ENV']
+            elif 'CONDA_PREFIX' in os.environ:
+                virtenv_dist = 'anaconda'
```

Comment:
`python_dist = 'anaconda'`

Reply:
p.s. I see that in one of my previous comments I made this error... (`virtenv_dist is 'default' or 'system'` is used only if `python_dist == 'default'` and `python_dist is 'default' or 'anaconda'`)